### PR TITLE
Bug 1806284: add await to create Image Stream request in deploy-image

### PR DESCRIPTION
**Pull request by sahil143**
This Pr fixes the intermittent failure of deploy image integration tests.

Failure message: `Deployment.apps \"hello-openshift\" is invalid: spec.template.spec.containers[0].image: Forbidden: this image is prohibited by policy: this image is prohibited by policy (changed after admission)`

### CODE REVIEW

File: frontend/integration-tests/tests/deploy-image.scenario.ts [hunk 1]
```diff
@@ -24,7 +24,6 @@ describe('Deploy Image', () => {
 
     it('should render applications dropdown disabled', async () => {
       // Navigate to the deploy-image page
-      await browser.get(`${appHost}/deploy-image/ns/${testName}?preselected-ns=${testName}`);
```
Comment (christianvogt) [inline frontend/integration-tests/tests/deploy-image.scenario.ts]: Why is this removed?
Also, the comment wasn't removed.

Comment (sahil143) [reply frontend/integration-tests/tests/deploy-image.scenario.ts]: In the first test, we are already navigating to deploy image form. This was causing the page to refresh again even though we are already on the deploy image form. https://github.com/openshift/console/pull/4474/files#diff-eb6a4820e46e49f73fc3bf67cc643894R16

Comment (sahil143) [reply frontend/integration-tests/tests/deploy-image.scenario.ts]: removed the comment

**Comment by christianvogt** [comment]
@spadgett i believe we were still seeing the issue even after this fix went in. It may have mitigated it somewhat though. Unsure without stats.

**Comment by spadgett** [comment]
> @spadgett i believe we were still seeing the issue even after this fix went in. It may have mitigated it somewhat though. Unsure without stats.

@adambkaplan Can you help us troubleshoot this error? We're creating an ImageStream and a Deployment with a trigger in parallel, and it seems to be racing.

> Deployment.apps \"hello-openshift\" is invalid: spec.template.spec.containers[0].image: Forbidden: this image is prohibited by policy: this image is prohibited by policy (changed after admission)

@christianvogt We might await the Deployment and *then* create the ImageStream. I think that would avoid the problem entirely.

**Comment by christianvogt** [comment]
@spadgett  best thing we can do is to make the change and try running locally many times to see if we can capture stats before and after before we re-instate the tests on master.
cc @sahil143
